event/events.go

```diff
@@ -211,3 +218,31 @@ func toCommentEvent(op types.Operation, commonEvent CommonEvent) Event {
 		}
 	}
 }
+
+// Delete Comment
+type DeleteCommentEvent struct {
+	CommonEvent
+	PermLink       string
+	Author         string
+}
+
+func (e DeleteCommentEvent) Type() Type {
+	return DeleteCommentEventType
+}
+
+func (e DeleteCommentEvent) Common() CommonEvent {
+	return e.CommonEvent
+}
+
+func toDeleteCommentEvent(op types.Operation, commonEvent CommonEvent) Event {
+	v, ok := op.(*types.DeleteCommentOperation)
+	if !ok {
+		panic(errWrongEventType)
```

Comment:
is it really good idea to panic in a library?

Reply:
This panic should never happen. If it happens the developer of the library made a mistake.